runtime/compiler/trj9/codegen/J9TreeEvaluator.cpp

```diff
@@ -48,7 +48,7 @@ void traceInstanceOfOrCheckCastProfilingInfo(TR::CodeGenerator *cg, TR::Node *no
       return;
       }
 
-   TR_AddressInfo * valueInfo = (TR_AddressInfo *)valueProfileInfo->getValueInfo(bcInfo, comp, TR_ValueProfileInfoManager::justInterpreterProfileInfo);
+   TR_AddressInfo * valueInfo = (TR_AddressInfo *)valueProfileInfo->getValueInfo(bcInfo, comp, AddressInfo, TR_ValueProfileInfoManager::justInterpreterProfileInfo);
```

Comment:
A static cast would be better here no?

Reply:
All API calls have been updated to use static casts.